pkg/workflows/wasm/host/wasm.go

```diff
@@ -5,27 +5,89 @@ import (
 	"errors"
 	"fmt"
 
+	"github.com/go-viper/mapstructure/v2"
 	"github.com/google/uuid"
-
 	"google.golang.org/protobuf/types/known/emptypb"
 
-	"github.com/smartcontractkit/chainlink-common/pkg/workflows/sdk"
-	wasmpb "github.com/smartcontractkit/chainlink-common/pkg/workflows/wasm/pb"
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities"
+	"github.com/smartcontractkit/chainlink-common/pkg/values"
+	legacySdk "github.com/smartcontractkit/chainlink-common/pkg/workflows/sdk"
+	legacywasmpb "github.com/smartcontractkit/chainlink-common/pkg/workflows/wasm/pb"
+	wasmpb "github.com/smartcontractkit/chainlink-common/pkg/workflows/wasm/v2/pb"
 )
 
-func GetWorkflowSpec(ctx context.Context, modCfg *ModuleConfig, binary []byte, config []byte) (*sdk.WorkflowSpec, error) {
+func GetTriggersSpec(ctx context.Context, modCfg *ModuleConfig, binary []byte, config []byte) ([]*wasmpb.TriggerSubscriptionRequest, error) {
```

Comment:
why the name change from workflow spec to trigger spec?

Reply:
It's not renamed; GetWorkflowSpec is lower in the file. I don't know why git didn't realize this was added before GetWorkflowSpec. I can put it lower in the file; maybe that'll reduce the diff by a line.